ansible/roles/spidermonkey/tasks/main.yml

```diff
@@ -19,19 +19,30 @@
   get_url:
     url: "http://ftp.mozilla.org/pub/mozilla.org/js/js185-1.0.0.tar.gz"
     dest: /tmp/js185-1.0.0.tar.gz
+
 - name: unpack Spidermonkey 1.8.5
   unarchive:
     src: /tmp/js185-1.0.0.tar.gz
     dest: /opt
     copy: no
+
+- name: patch source for debian9 build
+  patch:
+    src: "{{ role_path }}/files/fix-811665.patch"
+    basedir: /opt/js-1.8.5/
+    strip: 1
+  when: ansible_distribution == "Debian"
```

Comment:
This needs to be conditional to only apply this patch for the Debian 9 build, not all builds please.

Reply:
Done :+1: